assets/src/editor/events/dates-and-tickets-metabox/with-event-date-and-ticket-entity-filter-state.js

```diff
@@ -91,7 +91,7 @@ const withDateAndTicketEntityFilterState = createHigherOrderComponent(
 			}
 			return <WrappedComponent
 				allTicketEntities={ allTicketEntities }
-				ticketEntitiesLoaded={ ticketEntitiesLoaded }
+				loadingTicketEntities={ ! ticketEntitiesLoaded }
```

Comment:
Just a sanity check - have you verified this change in props is propagated throughout any children?  A simple thumbs up reaction is sufficient.

Reply:
yes in fact the child components were expecting an incoming prop named `loadingTicketEntities` but were not getting it.
I could have changed the children to use `ticketEntitiesLoaded ` instead, but this change makes it work exactly the same way as the corresponding date entity list and filter state HOC.